src/HealthGPS/analysis_module.cpp

```diff
@@ -307,29 +307,28 @@ DALYsIndicator AnalysisModule::calculate_dalys(Population &population, unsigned
 // NOLINTBEGIN(readability-function-cognitive-complexity)
 void AnalysisModule::calculate_population_statistics(RuntimeContext &context,
                                                      DataSeries &series) const {
-    auto min_age = context.age_range().lower();
-    auto max_age = context.age_range().upper();
     if (series.size() > 0) {
         throw std::logic_error("This should be a new object!");
     }
```

Comment:
I think it would make more sense to return a new `DataSeries` object rather than modify an input argument.

But that's not something we need to do now.